src/backend/gporca/libgpopt/src/translate/CTranslatorExprToDXL.cpp

```diff
@@ -460,7 +470,6 @@ CTranslatorExprToDXL::CreateDXLNode(CExpression *pexpr,
 	// for instance, if the grouping /order by /partition/ distribution columns
 	// are no longer needed
 	CDXLNode *pdxlnNew = NULL;
-
```

Comment:
restore this line

Reply:
Done

Reply:
I restored this line because clang-format pointed that  it shouldn't exist

Reply:
I don't think so, clang-format accepts current version with this line (without this patch)